platform-docs/docs/basic-concepts/settings.md

```diff
@@ -36,14 +65,6 @@ export const editorStyles = [
 		`,
 	},
 ];
-
-export default function App() {
-	return (
-		<BlockEditorProvider settings={ { styles: editorStyles } }>
```

Comment:
Non blocking question: would it be worth still including the `App` example in these, so that it's clear that `styles` is a key of the `settings` object, rather than a direct prop on `BlockEditorProvider`?

Reply:
Yeah, for now I kept it in the first setting and since I added a new initial setting, I moved it to the top. Open to reconsidering this, just leaving it for later.